master/internal/rm/multirm/multirm.go

```diff
@@ -415,6 +415,11 @@ func (m *MultiRMRouter) VerifyNamespaceExists(namespaceName string,
 	return nil
 }
 
+// MaybeRemoveNamespace is not supported.
+func (m *MultiRMRouter) MaybeRemoveNamespace(string) error {
```

Comment:
I thought resource quota project should work with multiRM too? so why is this func implemented for k8s rm but not multiRM?

Reply:
Yeah, you are right. Implemented for multiRM